pageserver/src/tenant/remote_timeline_client.rs

```diff
@@ -1134,12 +1139,14 @@ impl RemoteTimelineClient {
                     }
 
                     // sleep until it's time to retry, or we're cancelled
+                    let cancel = shutdown_token();
                     tokio::select! {
                         _ = task_mgr::shutdown_watcher() => { },
                         _ = exponential_backoff(
                             retries,
                             DEFAULT_BASE_BACKOFF_SECONDS,
                             DEFAULT_MAX_BACKOFF_SECONDS,
+                            &cancel
                         ) => { },
```

Comment:
Selection for the shutdown_watcher and shutdown_token now seems redundant, no?

Reply:
Yep, good spot fb54c09d34f3f884c9de1cb93cbe23731c3f603a